base/complex.jl

```diff
@@ -535,37 +535,39 @@ end
 #     return Complex(abs(iz)/r/2, copysign(r,iz))
 # end
 
-# compute exp(im*theta)
-function cis(theta::Real)
-    s, c = sincos(theta)
-    Complex(c, s)
-end
-
 """
-    cis(z)
+    cis(x)
 
-Return ``\\exp(iz)``.
+Same as `exp(im*x)`, this computes ``\\exp(i x)``, but is typically faster.
+The `cis` name is shorthand for Euler's formula: ``cos(x) + i sin(x) = \\exp(i x)``.
 
-See also [`cispi`](@ref), [`angle`](@ref).
+See also [`cispi`](@ref), [`sincos`](@ref), [`exp`](@ref), [`angle`](@ref).
 
 # Examples
 ```jldoctest
 julia> cis(π) ≈ -1
 true
 ```
 """
+function cis end
+function cis(theta::Real)
+    s, c = sincos(theta)
+    Complex(c, s)
+end
+
 function cis(z::Complex)
     v = exp(-imag(z))
     s, c = sincos(real(z))
     Complex(v * c, v * s)
 end
 
-cispi(theta::Real) = Complex(reverse(sincospi(theta))...)
-
 """
-    cispi(z)
+    cispi(x)
 
-Compute ``\\exp(i\\pi x)`` more accurately than `cis(pi*x)`, especially for large `x`.
+Equivalent to `exp(im*pi*x)`, this computes ``\\exp(i \\pi x)``, but is more accurate (especially for large `x`) and is typically faster.
```

Comment:
```suggestion
Similar to `exp(im*pi*x)`, this computes ``\\exp(i \\pi x)``, but is more accurate (especially for large `x`) and is typically faster.
```
It's not literally equivalent.

Reply:
what about `More efficient method for`

Reply:
I like it.